### PR TITLE
Register a content app in the startup class

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
In line with comment: https://github.com/umbraco/UmbracoDocs/pull/3638#pullrequestreview-811068459 I've created a PR: https://github.com/umbraco/Umbraco-CMS/pull/11691#issue-1062228629. In v9.2.0 we can register content apps in the `startup.cs` class.

### CODE REVIEW

**Comment by busrasengul** [comment]
Thank you @erikjanwestendorp 
We'll review this soon!

File: Extending/Content-Apps/index-vpre9.2.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+
+Content Apps can be configured to appear dependent on Section, Content Type and User Group Permissions.
+
+#### Read-Only
+
+Content Apps are designed to be companions to the Content Item. They should enhance the editor's experience by enabling quick access to contextual information for the particular content item they are editing. Content Apps are not intended to be used for the editing content.
+
+## Creating a Custom Content App
+
+This guide explains how to set up a custom Content App in the following steps:
```
Comment (sophie-neale) [inline Extending/Content-Apps/index-vpre9.2.md:44]:
```suggestion
This guide explains how to set up a Custom Content App in the following steps:
```
A little uppercase C for consistency 🙂

File: Extending/Content-Apps/index-vpre9.2.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+{
+    // array of files we want to inject into the application on app_start
+    "javascript": [
+        "~/App_Plugins/WordCounter/wordcounter.controller.js"
+    ]
+}
+```
+
+:::tip
+You can also have a coloured icon for your Content App by specifying the icon in the format `icon-[name of icon] color-[name of color]`. For eg, an indigo colored icon can be specified for your Content App by specifying the icon as  `"icon-calculator color-indigo"` in your Content App C# class or *package.manifest* .
```
Comment (sophie-neale) [inline Extending/Content-Apps/index-vpre9.2.md:275]:
```suggestion
You can also have a colored icon for your Content App by specifying the icon in the format `icon-[name of icon] color-[name of color]`. For eg, an indigo colored icon can be specified for your Content App by specifying the icon as  `"icon-calculator color-indigo"` in your Content App C# class or *package.manifest* .
```

Again a tiny tweak for consistency 👍🏼

**Review by sophie-neale — COMMENTED** [review]
Thanks Erik! I've had a quick scan before the Docs team gets together later this week and fixed a couple of typos for you ☺️

File: Extending/Content-Apps/index-vpre9.2.md (hunk 1 above)
Comment (erikjanwestendorp) [reply Extending/Content-Apps/index-vpre9.2.md:44]: @sophie-neale Thanks for your suggestion! 😄

File: Extending/Content-Apps/index-vpre9.2.md (hunk 2 above)
Comment (erikjanwestendorp) [reply Extending/Content-Apps/index-vpre9.2.md:275]: Thanks again @sophie-neale

**Comment by erikjanwestendorp** [comment]
@sophie-neale Thanks for you review! 😄 I just updated the PR.

**Review by marcemarc — APPROVED** [review]
Thanks Erik!

**Pull request merged**